diff -ruN a/vendor/lineage/release/aconfig/bp2a/Android.bp b/vendor/lineage/release/aconfig/bp2a/Android.bp
--- a/vendor/lineage/release/aconfig/bp2a/Android.bp	2025-10-03 09:30:43.469984450 +0800
+++ b/vendor/lineage/release/aconfig/bp2a/Android.bp	2025-10-03 09:31:14.207652144 +0800
@@ -17,5 +17,6 @@
     name: "aconfig_value_set-lineage-bp2a",
     values: [
     	"aconfig-values-bp2a-android.security-all",
+	"aconfig-values-bp2a-com.android.systemui-all",
     ],
 }
diff -ruN a/vendor/lineage/release/aconfig/bp2a/com.android.systemui/Android.bp b/vendor/lineage/release/aconfig/bp2a/com.android.systemui/Android.bp
--- a/vendor/lineage/release/aconfig/bp2a/com.android.systemui/Android.bp	1970-01-01 08:00:00.000000000 +0800
+++ b/vendor/lineage/release/aconfig/bp2a/com.android.systemui/Android.bp	2025-10-03 09:32:17.289971073 +0800
@@ -0,0 +1,21 @@
+// Copyright 2025 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+aconfig_values {
+  name: "aconfig-values-bp2a-com.android.systemui-all",
+  package: "com.android.systemui",
+  srcs: [
+    "*_flag_values.textproto",
+  ]
+}
diff -ruN a/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_compose_fragment_flag_values.textproto b/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_compose_fragment_flag_values.textproto
--- a/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_compose_fragment_flag_values.textproto	1970-01-01 08:00:00.000000000 +0800
+++ b/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_compose_fragment_flag_values.textproto	2025-10-03 09:33:00.068509860 +0800
@@ -0,0 +1,6 @@
+flag_value {
+  package: "com.android.systemui"
+  name: "qs_ui_refactor_compose_fragment"
+  state: ENABLED
+  permission: READ_ONLY
+}
diff -ruN a/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_flag_values.textproto b/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_flag_values.textproto
--- a/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_flag_values.textproto	1970-01-01 08:00:00.000000000 +0800
+++ b/vendor/lineage/release/aconfig/bp2a/com.android.systemui/qs_ui_refactor_flag_values.textproto	2025-10-03 09:33:35.318130179 +0800
@@ -0,0 +1,6 @@
+flag_value {
+  package: "com.android.systemui"
+  name: "qs_ui_refactor"
+  state: ENABLED
+  permission: READ_ONLY
+}

