--- a/frameworks/base/packages/SystemUI/src/com/android/systemui/screenrecord/ScreenRecordPermissionViewBinder.kt	2025-09-17 20:05:56.823151252 +0800
+++ b/frameworks/base/packages/SystemUI/src/com/android/systemui/screenrecord/ScreenRecordPermissionViewBinder.kt	2025-09-17 20:09:23.441531971 +0800
@@ -230,7 +230,7 @@
             else ScreenRecordingAudioSource.NONE
         val lowQuality = lowQualitySwitch.isChecked
         val longerDuration = longerDurationSwitch.isChecked
-        val hevc = hevcSwitch.isChecked
+        val hevc = false
         val startIntent =
             PendingIntent.getForegroundService(
                 userContext,

